src/Microsoft.NetCore.Analyzers/Core/Microsoft.Composition.Analyzers/MicrosoftCompositionAnalyzersResources.Designer.cs

```diff
@@ -40,7 +40,8 @@ internal class MicrosoftCompositionAnalyzersResources {
         internal static global::System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
-                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.Composition.Analyzers.MicrosoftCompositionAnalyzersResources", typeof(MicrosoftCompositionAnalyzersResources).GetTypeInfo().Assembly);
+                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.NetCore.Analyzers.Microsoft.Composition.Analyzers.MicrosoftCompositionA" +
```

Comment:
Yes, it seems to be auto-generated where it is appending the assembly name with custom tool namespace.